torch/utils/collect_env.py

```diff
@@ -165,7 +165,7 @@ def get_cudnn_version(run_lambda):
         # Use CUDNN_LIBRARY when cudnn library is installed elsewhere.
         cudnn_cmd = 'ls /usr/local/cuda/lib/libcudnn*'
     else:
-        cudnn_cmd = 'ldconfig -p | grep libcudnn | rev | cut -d" " -f1 | rev'
+        cudnn_cmd = 'ldconfig -p | grep libcudnn | rev | cut -d -f1 | rev'
```

Comment:
This does NOT look right!

Reply:
ahh, my mistake, thx
```suggestion
        cudnn_cmd = 'ldconfig -p | grep libcudnn | rev | cut -d" " -f1 | rev'
```